shifterator/plotting.py

```diff
@@ -280,32 +249,76 @@ def plot_contributions(ax, top_n, bar_dims, bar_colors, plot_params):
         `detailed`, and `bar_linewidth` are used
     """
     # Set plotting params
-    ys = range(1, top_n + 1)
-    alpha = plot_params['alpha_fade']
-    width = plot_params['bar_width']
-    linewidth = plot_params['bar_linewidth']
-    edgecolor = ['black'] * top_n # hack b/c matplotlib has a bug
-    if plot_params['detailed']:
+    bar_count = min(top_n, len(bar_dims["total_heights"]))
```

Comment:
In what situation is `bar_count` needed? Is it when there are less than `top_n` words? Does everything output correctly if there are less than `top_n` words? I haven't tried that situation before I think.

Reply:
Yup, this was just to make things look right when there weren't enough words. It should almost never get encountered in the wild, but some of my tiny test cases didn't look right without it.